dataverse-webapp/src/main/webapp/loginpage.xhtml

```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml"
       xmlns:h="http://java.sun.com/jsf/html"
-      xmlns:f="http://java.sun.com/jsf/core"
       xmlns:ui="http://java.sun.com/jsf/facelets"
       xmlns:o="http://omnifaces.org/ui"
       xmlns:p="http://primefaces.org/ui"
       xmlns:fn="http://java.sun.com/jsp/jstl/functions"
-      xmlns:jsf="http://xmlns.jcp.org/jsf">
+      xmlns:jsf="http://xmlns.jcp.org/jsf"
+      xmlns:f="http://xmlns.jcp.org/jsf/core"
+>
+
+<!--@elvariable id="OAuth2Page" type="edu.harvard.iq.dataverse.authorization.providers.oauth2.OAuth2LoginBackingBean"-->
```

Comment:
I don't think you need elvariable, the IDE already has direct reference to the backing bean page.

Reply:
Yeah, you're right. It was needed before I've added @Named. Should  work fine without it now.